front-end/src/macro/generic_step.jsx

```diff
@@ -76,13 +76,15 @@ const mapStateToProps = state => {
     macro: state.macros,
     lightings: state.lights,
     subsystem: [
-      { id: 'timers', name: 'timer' },
-      { id: 'phprobes', name: 'pH' },
-      { id: 'ato', name: 'ATO' },
-      { id: 'temperature', name: 'temperature' },
-      { id: 'lightings', name: 'lighting' },
-      { id: 'system', name: 'system' },
-      { id: 'doser', name: 'doser' }
+      // capabilities:..  are the subsytem names (plural or cathegory), corresponding to tab pages, whereas
+      // function:... are individual devices in these cathegories
```

Comment:
individual entities? Things like macro, timer are not devices.

Reply:
You're right, 'device' may not be the best term (non-native speaker ...).
The idea is to clearly distinguish the containers (visible as tabs and settings:capabilities) from the controller entities.

This may be the preparation to decouple containers from entities, such that the user can collect all his analog probes on one tab, and all things reading digital inputs on another, by setting the container name himself.
The difficulty here is the REST API, as this should use fixed paths I guess.